### DISCUSSION TITLE
Is it possible to trigger an event when user gets infront of a image?

### DISCUSSION THREAD

**Discussion by aqibshabbir** [ClementCariou/virtual-art-gallery#26] [Q&A]
Hi,

First of all i would say you have done amazing work, really appreciate you for making it open source.  
I have a question, I ll be glad if you could help me.

My question is when some user moves in-front or gets in a xyz range of a image, is it possible to trigger an event and alert() the title of the image or user could be prompt to press enter to get image title?

Note: This approach will be used to get the details of image using API and display them in modal. If you have some other technique please let me know.

Thanks in advance.

**Comment by ClementCariou** [comment]
The placement.js script exports a function called update which has access to the variable batch. This array contains the nearby paintings. You can simply check the distance between the player (the pos argument of update) and the paintings (the pos attribute of the painting object).
